app/utils/useMusicNavigation.js

```
@@ -0,0 +1,18 @@
+export function useMusicNavigation({ playlist, playlistIndex, onNext, onPrevious, audioRef, onSetCurrentTime, onPause }) {
+    const handlePrevious = () => {
```

ESLint

[error] Delete `··`

(prettier/prettier)